langchain/tools/image/utils.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+
+from steamship.data.workspace import SignedUrl
```

Comment:
For these, we should lazy import them to make them optional dependencies

Reply:
Resolved!